src/routes/blogs.rs

```diff
@@ -82,8 +87,11 @@ fn valid_slug(title: &str) -> Result<(), ValidationError> {
 }
 
 #[post("/blogs/new", data = "<form>")]
-pub fn create(conn: DbConn, form: LenientForm<NewBlogForm>, user: User, intl: I18n) -> Result<Redirect, Ructe> {
+pub fn create(form: LenientForm<NewBlogForm>, rockets: PlumeRocket) -> Result<Redirect, Ructe> {
     let slug = utils::make_actor_id(&form.title);
+    let conn = rockets.conn;
+    let intl = rockets.intl;
+    let user = rockets.user.unwrap();
```

Comment:
Maybe in another PR (because I think it is a bit out of scope for this one), we could replace all the "login required" routes with something like:

```rust
rockets.user.ok_or(Err(ErrorPage::Unauthorized("You need to be logged in to do this")))?;
```

(I don't remember if `ok_or` is the correct function to use here, but I think you got what I wanted to do)